scripts/mk_nuget_task.py

```diff
@@ -22,14 +22,14 @@ def mk_dir(d):
         os.makedirs(d)
 
 
-os_info = {"z64-ubuntu-14" : ('so', 'ubuntu.14.04-x64'),
-           'ubuntu-18' : ('so', 'ubuntu-x64'),
-           'ubuntu-20' : ('so', 'ubuntu-x64'),
-           'glibc-2.31' : ('so', 'ubuntu-x64'),
+os_info = {"z64-ubuntu-14" : ('so', 'linux-x64'),
```

Comment:
ubuntu-14 is probably long gone and I will remove on next occasion.
I use either 18 or default 20 these days. I understood there are some incompatibilities with 20 on downlevel platforms.